includes/abilities/class-migrate-abilities.php

```
@@ -132,28 +141,239 @@ public function execute_list_backups(): array {
+					$job = $body;
+					if ( 'done' === $body['state'] || 'error' === $body['state'] ) {
+						break;
+					}
+				}
+			}
+			sleep( 2 ); // phpcs:ignore WordPress.PHP -- poll cadence for a remote batch job.
+		}
+
+		$state = ( $job && isset( $job['state'] ) ) ? $job['state'] : 'unknown';
+		if ( 'error' === $state ) {
+			return new WP_Error( 'restore_failed', __( 'Destination reported an error during restore.', 'emcp-tools' ), array( 'job' => $job ) );
```

PHPStan (2.2.8)

[error] Class WP_Error constructor invoked with 3 parameters, 0-2 required.


(arguments.count)